app/src/script.js

```diff
@@ -147,35 +147,23 @@ async function getDelayedScript(scriptId) {
 
   if (executionTime === '0') return {}
 
-  let description = ''
   let executionTargets = []
+  let path = []
 
   try {
-    const path = await app.describeScript(evmCallScript).toPromise()
+    path = (await app.describeScript(evmCallScript).toPromise()) || []
 
     executionTargets = [...new Set(path.map(({ to }) => to))]
-
-    description = path
-      ? path
-          .map(step => {
-            const identifier = step.identifier ? ` (${step.identifier})` : ''
-            const app = step.name ? `${step.name}${identifier}` : `${step.to}`
-
-            return `${app}: ${step.description || 'No description'}`
-          })
-          .join('\n')
-      : ''
   } catch (error) {
     console.error('Error describing script', error)
-    description = 'Invalid script. The result cannot be executed.'
   }
 
   return {
     scriptId,
-    executionTime: marshallDate(executionTime),
-    executionDescription: description,
```

Comment:
Note that removing this here will likely cause old cached frontends to crash and require a state clear.

I was struggling with this in the Voting app as well, but I think we could leave the usage of `describePath(path)` in the script for now.

Reply:
hmm why that happens?

Reply:
Because the state from aragon.js is cached, and the new frontend should fallback to what was previously available.

Ideally we would purge the cache in aragon.js when we detect an app is updated... but it has a few dependencies on updating how the cache works :/